src/types.js

```diff
@@ -75,10 +75,7 @@ export type Axis = VerticalAxis | HorizontalAxis
 export type Placeholder = {|
   // We apply the margin separately to maintain margin collapsing
   // behavior of the original element
-  withoutMargin: {|
-    height: number,
-    width: number,
-  |},
+  paddingBox: Area,
```

Comment:
rather than just passing a cut I just pass the whole thing now. Because we name it `paddingBox` it is clearer that we are passing the right thing